charts/consul/values.yaml

```diff
@@ -2636,6 +2651,10 @@ meshGateway:
   # Requirements: consul 1.6.0+ if using `global.acls.manageSystemACLs``.
   enabled: false
 
+  # Override global log verbosity level for mesh-gateway-deployment pods. One of "trace", "debug", "info", "warn", or "error".
+  # @type: string
+  logLevel: ""
```

Comment:
Oh one last thing-- I think when we default to some higher level value, we typically use `"-"` rather than empty string as the default.

Reply:
I see this pattern mainly for boolean based fields. Newly introduced string based fields always have an empty string. DO we still need to change this?

Reply:
Yup you're right!